lib/services/elasticsearch.js

```diff
@@ -1433,40 +1550,47 @@ class ElasticSearch extends Service {
   {
     const
       query = { ids: { values: [] } },
-      toGet = [],
+      validIds = [],
       partialErrors = [];
 
     // @todo should be done in clientAdapter
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)